src/ape/api/accounts.py

```diff
@@ -200,12 +200,12 @@ def deploy(self, contract: "ContractContainer", *args, **kwargs) -> "ContractIns
         contract_name = contract.contract_type.name or "<Unnamed Contract>"
         logger.success(f"Contract '{contract_name}' deployed to: {address}")
 
-        from ape import _converters
+        from ape import converters
         from ape.contracts import ContractInstance
 
         return ContractInstance(  # type: ignore
             _provider=self.provider,
-            _converter=_converters,
+            _converter=converters,
```

Comment:
should try to DI `converters`, and we've talked about removing the need for `_provider=...` by injecting `network_manager` into `AddressAPI` instead and having `provider` either be a `property` (for `AccountAPI`) or `cached_property` (for `ContractInstance`) tied to `NetworkManager.active_provider`